src/ui/proxy/interceptors.go

```diff
@@ -51,8 +54,8 @@ func MatchPullManifest(req *http.Request) (bool, string, string) {
 type policyChecker interface {
 	// contentTrustEnabled returns whether a project has enabled content trust.
 	contentTrustEnabled(name string) bool
-	// vulnerableEnabled  returns whether a project has enabled content trust.
-	vulnerableEnabled(name string) bool
+	// vulnerableEnabled  returns whether a project has enabled vulnerable, and the project's severity.
+	vulnerableEnabled(name string) (bool, string)
```

Comment:
better rename the function to vulnerablePolicy()?  xxxEnabled is clearly a func that returns bool.